src/index.js

```diff
@@ -295,8 +298,18 @@ const snyker = async () => {
   console.log("[SNYKER: STARTING]");
 
   MAX_RETRIES = argv.retries || DEFAULT_RETRIES;
+  PRESERVE_INTEGRITY = argv["preserve-integrity"] || false;
+
+  // We need to determine whether we're using Yarn or NPM
+  // Prioritise "lockfile" flag, then check for yarn.lock, then package-lock.json
+  // If none of these files exist, default to yarn.lock
+  const lockFileName =
```

Comment:
could consider `npm-shrinkwrap.json` and `pnpm-lock.yaml` too

Reply:
I think these would be good for a future change/contribution 👍